bar-trender-front-end/src/App.js

```diff
@@ -7,44 +7,20 @@ export default class App extends React.Component {
   state = {
     loading: true,
     qr: null,
-    error: null,
   };
   
-
   async componentDidMount()
   {
-
-    var token = sessionStorage.getItem("token");
-    const id_establishment = this.props.idEstablishment;
-    const id_discount = this.props.idDiscount;
-    
-    if(!token){
-      this.setState({error: "Necesitas haber iniciado sesión para poder ver el descuento"})
-    }else{
-      const url = 'https://develop-backend-sprint-01.herokuapp.com/v1/establishments/'+id_establishment+'/discounts/'+id_discount+'/getQR';
-      const response = await fetch(url, {
-        method: 'POST',
-          headers: {
-            'Content-Type': 'application/json',
-            'token': token,
-          },
-          body: JSON.stringify({redirect_url:'www.google.es'})
-      });
-      if(response.status == 200){
-        const data = await response.blob();
-        this.setState({qr: URL.createObjectURL(data), loading: false});
-      }else{
-        const data = await response.json();
-        this.setState({loading: false, error: data.error});
-      }
-    }
+    const url = 'http://localhost:8000/main/test/';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7164324)
```suggestion
    const url = "http://localhost:8000/main/test/";
```